lib/internal/process.js

```diff
@@ -283,6 +283,39 @@ function setupUncaughtExceptionCapture(exceptionHandlerState) {
   };
 }
 
+function setupCompareVersion() {
+  const {
+    majorVersion,
+    minorVersion,
+    patchVersion,
+    prereleaseTag,
+  } = process.release;
+
+  process.release.compareVersion =
+    (major, minor, patch, prerelease = false) => {
```

Comment:
I think the default value for the last parameter here can be removed as it's not necessary and will not affect tag comparisons when it is `undefined` instead of `false`.